docs/zh_cn/community/contributing.md

```diff
@@ -144,9 +145,9 @@ git push -u origin {branch_name}
 (1) 在 GitHub 的 Pull request 界面创建拉取请求
 <img src="https://user-images.githubusercontent.com/27466624/204302289-d1e54901-8f27-4934-923f-fda800ff9851.png" width="1200">
 
-(2) 根据指引修改 PR 描述，以便于其他开发者更好地理解你的修改
+(2) 根据指引修改 PR 描述，以便于其他开发者更好地理解你的修改（注意在PR branch左侧的base需要修改为dev分支）
```

Comment:
There should be a space between (English characters, numbers) and Chinese characters.